JASP-Tests/R/tests/testthat/test-anovarepeatedmeasuresbayesian.R

```diff
@@ -90,7 +90,7 @@ test_that("Effects table results match", {
   options$effects <- TRUE
 
   refTables <- list(
-    allModels = list("<unicode>", "RM_FACTOR_1", 0.4, 0, 0.6, 1, 2.08122183080571,
+    allModels = list(1000799917193443, "RM_FACTOR_1", 0.4, 0, 0.6, 1, 2.08122183080571,
```

Comment:
Maybe you could use less extreme data so this will pass on different operating systems?

Reply:
I didn't change the input data for this test, but I will now. If you don't mind I'll merge the test for the effects table and those for the main table since the two are independent (and the unit tests timed out a couple of times already).